README.md

```diff
@@ -1,5 +1,7 @@
 ![LucidGloves](https://cdn.discordapp.com/attachments/785135646082990120/873666310855360582/LucidGlovesBlackwbkgnd.png)
 
+**[English](https://github.com/RyanPiao/lucidgloves/blob/main/README.md) | [简体中文](https://github.com/RyanPiao/lucidgloves/blob/main/README_CN.md)**
```

Comment:
These need to be updated as they link to your fork and not the master repo.

Also, please move this line to the bottom of README.md.